crates/prune/src/pruner.rs

```diff
@@ -226,6 +234,14 @@ impl<DB: Database> Pruner<DB> {
         self.metrics.duration_seconds.record(elapsed);
 
         trace!(target: "pruner", %tip_block_number, ?elapsed, "Pruner finished");
+        info!(
+            target: "pruner",
+            %tip_block_number,
+            %done,
+            "Pruner finished after {:?} nanosec, pruned {} entries",
+            elapsed.subsec_nanos(),
+            total_pruned_entries
+        );
```

Comment:
Couple things:
1. As @joshieDo said, we want to log total number of pruned entries, and not the number of pruned parts. It would be nice to show the summary of pruning, i.e. number of entries per part. So each prune part will have its own field.
2. Total elapsed time is nice, but let's put it into a field instead of the log message. Log message should be a human-readable text, and all the additional data should go to fields.
3. New `info` log now replicates the `trace` one, and since `INFO < TRACE` in terms of log levels, we can remove the old `trace` log.

Reply:
Points 2 and 3 should be resolved, I'm not very familiar with the concept of pruned entries inside pruned parts because I don't know this part very well, can you give me an example of how to calculate that I imagine to place it in a `HashMap` which will summarize pruned entries sorted by pruned parts?

Reply:
yes, that's right. But I just realized it might a bit tricky because account and storage history pruning methods have two internal counters for pruned entries, so it's not clear which one to put in the log.

Let's just have a separate field for each of pruning parts with `done` value returned by the pruning method? So, something like 
```rust
info!(
    target: "pruner",
    %tip_block_number,
    ?elapsed,
    %done,
    ?parts_done,
    "Pruner finished"
);
```
where `parts_done` is a `HashMap<PrunePart, bool>`.